tests/index.test.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable no-undef */
+import chai, { expect } from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../server/app';
+
+chai.use(chaiHttp);
+
+describe('HOMEPAGE', () => {
+  it('should respond with welcome message', done => {
+    chai
+      .request(app)
+      .get('/')
+      .end((err, res) => {
+        expect(res).to.have.status(200);
+        done();
+      });
+  });
+});
+
+export { chai, expect, app };
```

Comment:
Why are you exporting `chai`, `expect` and `app`?

Reply:
we have an index file for our test. I exported chai, expect and the app so for each of our test files we will import from the index file and use it. That was the setup we tried to explain earlier today explained

Reply:
But you don't need to export `chai`, `expect` and `app` from this file. If they are needed in any other test file we can easily import it directly can't we? Like so:

```
import chai, { expect } from 'chai';
import app from './pathToApp.js
```

I think the use of the index file in your scenario is useful when you have a bunch of functions from different files that you want to export from one file.

In the case of our tests, I don't think you need an index file for the tests instead you could have different tests to test different parts of the app e.g. server, controllers, etc.

Do you understand?

Reply:
Yes, I understand that. I'll make the necessary correction.